### PR TITLE
addition add methods

### PR DESCRIPTION
**Pull request by plukevdh**
some useful helpers to allow flags to be predefined and passed in _en masse_ to a command.

### CODE REVIEW

File: src/commander/flags.cr [hunk 1]
```diff
@@ -11,7 +11,15 @@ class Commander::Flags
     flags.each { |flag| yield flag }
   end
 
-  def add(flag)
+  def add(*multi)
```
Comment (mrrooijen) [inline src/commander/flags.cr]: Rename `*multi` to `*flags` for clarity.

File: src/commander/flags.cr [hunk 2]
```diff
@@ -11,7 +11,15 @@ class Commander::Flags
...
+    add multi.to_a
+  end
+
+  def add(multi : Array(Commander::Flag))
```
Comment (mrrooijen) [inline src/commander/flags.cr]: Rename `multi` to `flags` for clarity.

**Comment by mrrooijen** [comment]
lgtm after minor changes, thanks! :+1:

File: src/commander/flags.cr (hunk 2 above)
Comment (plukevdh) [inline src/commander/flags.cr]: The only reason I didn't here is that it "clashes" with the getter. I think I originally used the getter then refactored. I'll update.

Comment (mrrooijen) [inline src/commander/flags.cr]: I figured. ;) 

But I'm fine with using `flags` over `multi` since the scope is so small it won't be confusing.

**Comment by plukevdh** [comment]
Done and squashed.

**Comment by mrrooijen** [comment]
Thanks! Released v0.3.1.

**Comment by plukevdh** [comment]
:metal:

**Pull request merged**